docs2/site/docs/migrations/migration5.md

```diff
@@ -82,6 +82,42 @@ a list or array of a single item. For example, `{"query":"{ hero }"}` deserializ
 serializing objects to and from `string` values. For the `GraphQL.SystemTextJson`
 and `GraphQL.NewtonsoftJson` libraries, these serialize and deserialize to JSON strings.
 
+### 6. `AutoRegisteringObjectGraphType` and `AutoRegisteringInputObjectGraphType` enhancements
+
+#### Overridable base functionality
+
+The classes can be overridden, providing the ability to customize behavior of automatically
+generated graph types. For instance, to exclude properties marked with a custom attribute
+called `[InternalUse]` you could write this:
+
+```csharp
+private class CustomAutoObjectType<T> : AutoRegisteringObjectGraphType<T>
+{
+    protected override IEnumerable<FieldType> CreateFieldTypeList()
```

Comment:
CreateFieldTypeList -> ProvideFields ?

Reply:
I mean I don't like `list` in name a bit.

Reply:
Sure